website/blog/2021-12-13-zero-day-vulnerability.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Security Warning: New Zero-day Log4J2 Vulnerability"
```

Comment:
Lets change the title of this blog and the file name to include "Mitigating Log4J 2 Vulnerability-cve-2021-44228